commit 2c685d71a6310aa37f1ed1fe02598d0d309624e0 (HEAD -> master)
Author: John Levon <john.levon@joyent.com>
Date:   2019-10-07T15:04:31+00:00 (8 hours ago)
    
    OS-8006 update gcc to 7.4.0-il-1
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/gcc7/Makefile b/gcc7/Makefile
index 889bce8..780fc95 100644
--- a/gcc7/Makefile
+++ b/gcc7/Makefile
@@ -14,19 +14,17 @@
 #
 
 #
-# The GCC tarball is the gcc-7.3.0-il-3 release tag of the
+# The tarball is from a specific tagged release of
 # https://github.com/illumos/gcc repository.  This includes all of the
-# illumos-specific modifications we need.  We could probably do a git
-# clone instead, but the tarball method is easier given these Makefiles.
-# The reason that there are two gcc entiries below is because the first
-# is used by github to indicate the repository and the second is part of
-# the tag.
+# illumos-specific modifications we need.  We could probably do a git clone
+# instead, but the tarball method is easier given these Makefiles.  The odd
+# prefix of "gcc-gcc-..." is due to github's <repo>-<tag> naming.
 #
 
 include ../Makefile.defs
 
 GCC_VER = 7
-VER = gcc-gcc-7.3.0-il-3
+VER = gcc-gcc-7.4.0-il-1
 MPFR_VER = mpfr-3.1.6
 GMP_VER = gmp-6.1.2
 MPC_VER = mpc-1.0.3
diff --git a/gcc7/gcc-gcc-7.3.0-il-3.tar.gz.sha1 b/gcc7/gcc-gcc-7.3.0-il-3.tar.gz.sha1
deleted file mode 100644
index bb70756..0000000
--- a/gcc7/gcc-gcc-7.3.0-il-3.tar.gz.sha1
+++ /dev/null
@@ -1 +0,0 @@
-46e54bbe041cefd8395574d9fb686efecdc7ebce
diff --git a/gcc7/gcc-gcc-7.4.0-il-1.tar.gz.sha1 b/gcc7/gcc-gcc-7.4.0-il-1.tar.gz.sha1
new file mode 100644
index 0000000..ca7920e
--- /dev/null
+++ b/gcc7/gcc-gcc-7.4.0-il-1.tar.gz.sha1
@@ -0,0 +1 @@
+d766e6d21653d865f148bee5b80ae692d46c3d25
